cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1126,6 +1126,22 @@
       },
       "Condition": "CreateEFSSubstack"
     },
+    "CloudWatchLogGroup": {
+      "Type": "AWS::Logs::LogGroup",
+      "Properties": {
+          "LogGroupName": {
+            "Fn::Sub": [
+              "/aws/${cluster_name}",
+              {
+                "cluster_name": {
+                  "Ref": "AWS::StackName"
```

Comment:
This is `parallelcluster-[cluster_name]` not the cluster name. You can Fn::Split to get just the cluster name. Search the code for examples.

Reply:
That was on purpose. The intent was to have something in the log group's name signifying that it was a cluster resource.

Reply:
I'd prefer `/aws/parallelcluster/[cluster_name]` since it follows similar services, such as:
* `/aws/batch/job`
* `/aws/lambda/[lambda_name]`
* `/aws/apigateway/[api_name]`
* `/aws/codebuild/[project_name]` 

So it seems like it's the standard format. I know it's much messier to implement in the cfn template.

Reply:
should be good to go now